app/models/scheme.rb

```diff
@@ -219,7 +219,11 @@ def validate_owning_organisation
   end
 
   def available_from
```

Comment:
there might be a more succinct way of writing this, see https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1396/files#diff-21a9bef9fe959a51cb7dc2afeb12b6ebafa4b89436172a40997c48f5d44e7813

quite possibly not though since we're anchoring everything at created_at